src/components/product-progress/step-progress-extractor.service.ts

```diff
@@ -60,12 +60,17 @@ const parseProgressRow =
       const percentDecimal = cell.asNumber;
       return percentDecimal ? percentDecimal * 100 : undefined;
     };
-    const steps = entries(stepColumns).map(
-      ([step, column]): StepProgressInput => ({
-        step,
-        completed: progress(column),
-      })
-    );
+    const steps = entries(stepColumns).flatMap(([step, column]) => {
```

Comment:
```suggestion
    const steps = entries(stepColumns).flatMap<StepProgressInput>(([step, column]) => {
```
Connect this type explicitly so we know this is the shape we are defining. Also helps with looking up usage & refactors.